src/layouts/Sites.jsx

```diff
@@ -11,6 +12,33 @@ import { SITES_IS_PRIVATE_KEY } from "constants/constants"
 import elementStyles from "styles/isomer-cms/Elements.module.scss"
 import siteStyles from "styles/isomer-cms/pages/Sites.module.scss"
 
+const Site = ({ siteNames }) => {
+  if (siteNames && siteNames.length > 0)
+    return siteNames.map((siteName) => (
```

Comment:
Nit: should be `Sites` since this component encompasses all sites

Reply:
good catch! as there's already a `Sites` component below, i'll rename them both to `Sites`/`SitesWrapper`!